x/wasm/client/cli/tx.go

```diff
@@ -62,11 +61,9 @@ func StoreCodeCmd(cdc *codec.Codec) *cobra.Command {
 				return err
 			}
 
-			// limit the input size
```

Comment:
Why is this check removed?

Otherwise, the function looks good

Reply:
Nevermind... I saw this is now handled by `ValidateBasic`. 
Very proper to remove it here to avoid duplication.